server/src/optima/dbmodels.py

```diff
@@ -46,7 +46,9 @@ class ProjectDb(db.Model):
                                 uselist=False)
     project_data = db.relationship('ProjectDataDb', backref='projects',
                                 uselist=False)
-    creation_time = db.Column(db.DateTime(timezone=True), server_default=text('now()'))
+    creation_time = db.Column(db.DateTime(timezone=True), default=db.func.now())
```

Comment:
The change for `creation_time` it's for consistency reasons, the effect of both is same and I think the later it's more robust because allows defaults on INSERT and UPDATE. Maybe only we could add to the migration script drop defaults
```sql
ALTER COLUMN creation_time DROP DEFAULT;
```
The defaults for `updated_time` it's not problematic, we could remove, but still it's no source of confusion since same `creation_time` and `updated_time` means no update.